### PR TITLE
Update changelog with latest devon_rex updates

### PR DESCRIPTION
**Pull request by radomirbosak**
These PRs are now included in the changelog
image

### CODE REVIEW

**Review by gnakagaw — APPROVED** [review]

**Comment by radomirbosak** [comment]
I need to check if this issue is not a problem
https://github.com/sider/devon_rex/issues/564
(klint compatibility with Java)

**Comment by radomirbosak** [comment]
Also, new version  heading needs to be added to the changelog.

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,8 +6,18 @@ All notable changes to this project will be documented in this file.
 
 [Full diff](https://github.com/sider/devon_rex/compare/2.45.11...HEAD)
 
+## 2.46.0
+
+[Full diff](https://github.com/sider/devon_rex/compare/2.45.11...2.46.0)
```
Comment (radomirbosak) [inline CHANGELOG.md:11]: > __[remark-lint]__ Link to https://github.com/sider/devon_rex/compare/2.45.11...2.46.0 is dead ([view](https://sider.review/gh/repos/200181683/pulls/704#issue-6229363))
>
> <details>
>
> | Rule | Severity |
> | ---- | -------- |
> | `no-dead-urls` | `warn` |
>
> You can [close this issue](https://sider.review/gh/repos/200181683/pulls/704#issue-6229363) if no need to fix it. [Learn more](https://help.sider.review/getting-started/working-with-issues).
>
> </details>

Tag will be created after merging this PR

**Comment by radomirbosak** [comment]
Note that the java update 15 -> 17 was reverted in the end in
https://github.com/sider/devon_rex/pull/705

due to [unclear support](https://github.com/pinterest/ktlint/issues/1195) in ktlint

**Pull request merged**